BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load(
+    "@bazel_gazelle//:def.bzl",
+    "gazelle",
+)
+
+# gazelle:prefix github.com/google/go-jsonnet
+gazelle(
+    name = "gazelle",
+)
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "builtins.go",
```

Comment:
hmmm... do we need to specify all these files explicitly? I thought gazelle can take care of dependencies. And I guess it wouldn't be possible to take only a subset of files from a Go package.

(I don't know bazel very well, just asking)

Reply:
Or is it synced automatically with the commands like `bazel run //:gazelle`?

Reply:
Yes, these files are written _primarily_ by Gazelle. There are a few places where I had to go in and adjust something Gazelle had proposed; you can see the occasional "#keep" comment on those lines. Past that, though, the intention is that us humans shouldn't be maintaining these files. We write the Go code as we please, and occasionally we have to run Gazelle to synchronize the Bazel files with our Go code.

There is [a program called _Autogazelle_](https://github.com/bazelbuild/bazel-gazelle/tree/master/cmd/autogazelle) that can do this for you by wrapping itself around some of the Bazel commands. I have not yet tried it yet, as I haven't found invoking Gazelle myself to be sufficiently onerous.